api/index.md

```diff
@@ -1,9 +1,9 @@
 ---
 uid: microsoft.quantum.standardlibsintro
```

Comment:
Is this UID still accurate, given that this page is for the API listing?

Reply:
Good point. I changed to microsoft.quantum.apiref-intro, and updated the three other files that referenced it.